docs/organizer-guide/confs/website.rst

```diff
@@ -0,0 +1,293 @@
+  You may need to update some reference to specific locations/years,
+  and keep an eye on any FIXME's or TODO's.
+* ``include/conf/`` contains text snippets which are generally the same
+  between conferences, or at least between conferences in the same location.
+  Examples include the description of lightning talks, or the Portland hike.
+* ``_templates/<year>/`` and ``_templates/include/`` contain HTML or RST
+  templates used in rendering the conference pages. This includes the
+  website design, navigation and snippets used in schedule rendering.
+
+The YAML files are validated using the schemas in ``_data/schema-*``.
+
+
+Typical conference workflow
+---------------------------
+
+* The initial conference site is created, with ``flaglanding`` typically
+  set to publish a simple landing page with the conference dates.
+* At some point, the content for the conference is filled in and verified,
+  like sponsorship prices. ``flaglanding`` is now switched off and remains off.
+* As soon as tickets go on sale, ``flagticketsonsale`` is switched on.
+* When the CFP opens, ``flagcfp`` is switched on. Once the CFP has closed,
+  it is switched off and remains off. The CFP details should be in the
+  general conference config file, under ``cfp``.
+* When speakers are announced, ``flagspeakersannounced`` is switched on
+  and remains on. The sessions should be in the
+  ``_data/<shortcode>-<year>-sessions.yaml`` file.
+* When the schedule is ready, ``flaghasschedule`` is switched on and remains on.
+  The schedule should be in the ``_data/<shortcode>-<year>-schedule.yaml`` file.
+* At some point the conference sells out. ``flagticketsonsale`` is switched off,
+  ``flagsoldout`` is switched on.
+* During the conference, if live streaming is available, ``flaglivestreaming``
+  is switched on. It is switched off after the conference ends.
+* After the conference ends, ``flagpostconf`` is switched on and remains on.
+* When videos are published on the Write the Docs website, ``flagvideos``
+  is switched on and remains on.
+
+Each time the state is updated, check the top/bottom buttons to ensure they
+still fit. For example, once tickets have sold out, the button "buy a ticket"
+should be removed.
+
+
+General conference config file
+------------------------------
+
+If you're setting up a new conference, the easiest may be to take a YAML
+config file from the previous conference, and adjust it.
+
+The items in the general conference config file are:
+
+* ``name``: the dynamic part of the name of the conference, e.g. ``Portland``
+  for Write the Docs Portland.
+* ``shortcode``: the shortcode as used in file paths, config names, and other
+  references to the conference, together with the year. Currently used are
+  ``portland``, ``prague``, and ``australia``.
+* ``year``: the conference year, e.g. ``2020``.
+* ``city``: the city for the conference.
+* ``local_area``: the state or country for the conference.
```

Comment:
For example, "Oregon" or "Czech Repubilc"? Does this need to be properly capitalized?